src/raftstore/store/fsm/store.rs

```diff
@@ -458,11 +470,11 @@ impl<'a, T: Transport, C: PdClient> StoreFsmDelegate<'a, T, C> {
     fn start(&mut self, store: metapb::Store) {
         if self.fsm.store.start_time.is_some() {
             panic!(
-                "{} unable to start again with meta {:?}",
-                self.fsm.store.tag, store
+                "[store {}] unable to start again with meta {:?}",
```

Comment:
I don't think this can be correctly parsed.

Additionally meta should be a field?

Reply:
It's a panic not a log.